src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.keys(robot);
+
+  for (let i = 0; i <= keys.length; i++) {
+    for (let j = i + 1; j <= keys.length; j++) {
+      if (robot[keys[i]] === robot[keys[j]]) {
+        return null;
+      }
+    }
+  }
```

Comment:
Do you really need two nested loops to check it? (and three loops at all)
I think only one iteration of robot keys is enough.
Let's try to optimize your solution.
You can just check it newRobot has properties from robot